setup.py

```diff
@@ -46,7 +48,6 @@
 EXTRAS_REQUIRE = {
```

Comment:
My doc building exercise failed without `m2r`. Also had to install `pandoc` (I had to install pandoc with conda rather than pip for some reason in my virtual environment...)

Reply:
Now I see we already have a note about pandoc. But should it also be in setup.py? Or is it technically a sub-requirement?

Reply:
Ah, I removed `m2r` from `setup.py` but not `conf.py`.  I'll fix that, good catch.

pandoc is used internally by `nbsphinx` for the notebook to rst conversion, so technically it's an indirect requirement, but it's on us to provide it to nbsphinx so it's not really black and white.  Is it even possible to include conda packages in setup.py?

Reply:
pandoc is on pypi, but for whatever reason installing it with pip in the virtual environment doesn't allow me to build the docs. I vote we continue to leave it out of setup.py since it is indirect.